README.md

```diff
@@ -11,189 +12,67 @@ This project explores the use of large language models (LLMs) for product recomm
 - **Sequential Recommendation System**: Predicts the next (maximum) 10 items a user might purchase in the future, based on their review history.
 - **Custom Data Processing**: Handles temporal data (order by time to prevent data/target/information leakage) and processes the sequence of user interactions, ensuring proper padding and handling of missing data.
 - **Fine-tuning GPT-2**: Fine-tuned GPT-2 medium model to handle recommendation tasks using user reviews, product metadata, and timestamps.
-- **Evaluation Metrics**: Supports evaluation with metrics such as Recall, Precision, MRR and more.
+- **Evaluation Metrics**: Supports evaluation with metrics such as Recall, Precision, MRR, HR@10, and NDCG@10.
```

Comment:
Please use the full names of the evaluation metrics instead of abbreviation for HR@10 and NDCG